src/Parser/PotreeBinParser.js

```diff
@@ -1,61 +1,117 @@
 import * as THREE from 'three';
 import PointsMaterial from '../Renderer/PointsMaterial';
 
+// See the different constants holding ordinal, name, numElements, byteSize in PointAttributes.cpp in PotreeConverter
+// elementByteSize is byteSize / numElements
+const POINT_ATTTRIBUTES = {
+    POSITION_CARTESIAN: {
+        numElements: 3,
+        arrayType: Float32Array,
+        attributeName: 'position',
+    },
+    COLOR_PACKED: {
+        numElements: 4,
+        arrayType: Uint8Array,
+        attributeName: 'color',
+        normalized: true,
+    },
+    INTENSITY: {
+        numElements: 1,
+        numByte: 2,
```

Comment:
Maybe you should add a comment to explain why you need this? (AFAIU it's because `Float16Array` doesn't exist so you emulate it)